docs/docs/stack_config.md

```diff
@@ -117,6 +118,10 @@ A dictionary of [CloudFormation Tags](https://docs.aws.amazon.com/AWSCloudFormat
 
 The ARN of a [CloudFormation Service Role](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-servicerole.html) that is assumed by CloudFormation to create, update or delete resources.
 
+### notifications
+
+List of SNS topic ARNs to publish stack related events to. A maximum of 5 ARNs can be specified per stack.
```

Comment:
Could we have a link to the AWS documentation?

Reply:
Could we also list which Sceptre actions (create, update, create-change-set) this parameter will affect?

Reply:
These are the only links I can find, but they aren't great:
1. http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html (would we have to add a link for create/update/delete?)
2. http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-add-tags.html (kind of vague and not helpful)

Do you have a preference?

Reply:
Ah yeah, not great - and no page anchors either. I'd say first option - but more explicit in referencing the documentation e.g. More information about stack notifications can found under the relevant section in the [AWS CloudFormation API documentation](http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html).